dumb-init.c

```diff
@@ -89,7 +92,11 @@ void forward_signal(int signum) {
 */
 void handle_signal(int signum) {
     DEBUG("Received signal %d.\n", signum);
-    if (signum == SIGCHLD) {
+
+    if (signal_temporary_ignores[signum] == 1) {
+        DEBUG("Ignoring signal %d during its first receive.\n", signum);
```

Comment:
maybe `Ignoring tty hand-off signal...` so it's more clear _why_ this is happening here